src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -74,21 +63,53 @@ def show_more(notifications)
 
   def fetch_notifications
     notifications = policy_scope(Notification).for_web.includes(notifiable: [{ commentable: [{ comments: :user }, :project, :bs_request_actions] }, :bs_request_actions, :reviews])
```

Comment:
Make a small PR to get rid of this cruft...

https://github.com/openSUSE/open-build-service/pull/16125